### PR TITLE
Suppress AWS maintenance mode message

### PR DESCRIPTION
**Pull request by jonatanschroeder**
As discussed on Slack, and based on discussion in https://github.com/aws/aws-sdk-js/issues/4354#issuecomment-1450660391.

### CODE REVIEW

**Comment by SethPoulsen** [comment]
This doesn't work for me. In fact, it somehow makes things worse. I get this on startup:

```
Starting PrairieLearn...
Trace
    at /PrairieLearn/node_modules/aws-sdk/lib/maintenance_mode_message.js:29:11
    at Object.<anonymous> (/PrairieLearn/node_modules/aws-sdk/lib/maintenance_mode_message.js:34:3)
    at Module._compile (node:internal/modules/cjs/loader:1191:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1245:10)
    at Module.load (node:internal/modules/cjs/loader:1069:32)
    at Function.Module._load (node:internal/modules/cjs/loader:904:12)
    at Module.require (node:internal/modules/cjs/loader:1093:19)
    at Module.Hook._require.Module.require (/PrairieLearn/node_modules/require-in-the-middle/index.js:101:39)
    at require (node:internal/modules/cjs/helpers:108:18)
    at Object.<anonymous> (/PrairieLearn/node_modules/aws-sdk/lib/core.js:88:1)
(node:50) NOTE: We are formalizing our plans to enter AWS SDK for JavaScript (v2) into maintenance mode in 2023.

Please migrate your code to use AWS SDK for JavaScript (v3).
For more information, check the migration guide at https://a.co/7PzMCcy
(Use `node --trace-warnings ...` to show where the warning was created)
NOTE: We are formalizing our plans to enter AWS SDK for JavaScript (v2) into maintenance mode in 2023.

Please migrate your code to use AWS SDK for JavaScript (v3).
For more information, check the migration guide at https://a.co/7PzMCcy
    at emitWarning (/PrairieLearn/node_modules/aws-sdk/lib/maintenance_mode_message.js:21:13)
    at /PrairieLearn/node_modules/aws-sdk/lib/maintenance_mode_message.js:32:5
    at Object.<anonymous> (/PrairieLearn/node_modules/aws-sdk/lib/maintenance_mode_message.js:34:3)
    at Module._compile (node:internal/modules/cjs/loader:1191:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1245:10)
    at Module.load (node:internal/modules/cjs/loader:1069:32)
    at Function.Module._load (node:internal/modules/cjs/loader:904:12)
    at Module.require (node:internal/modules/cjs/loader:1093:19)
    at Module.Hook._require.Module.require (/PrairieLearn/node_modules/require-in-the-middle/index.js:101:39)
    at require (node:internal/modules/cjs/helpers:108:18)
[nodemon] 2.0.20
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: js,json,sql
[nodemon] starting `node server.js`
[nodemon] 2.0.20
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): workspace_host/**/* lib/**/* sprocs/**/* prairielib/**/*
[nodemon] watching extensions: js,json,sql
[nodemon] starting `node workspace_host/interface.js`
false
Trace
    at /PrairieLearn/node_modules/aws-sdk/lib/maintenance_mode_message.js:29:11
    at Object.<anonymous> (/PrairieLearn/node_modules/aws-sdk/lib/maintenance_mode_message.js:34:3)
    at Module._compile (node:internal/modules/cjs/loader:1191:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1245:10)
    at Module.load (node:internal/modules/cjs/loader:1069:32)
    at Function.Module._load (node:internal/modules/cjs/loader:904:12)
    at Module.require (node:internal/modules/cjs/loader:1093:19)
    at require (node:internal/modules/cjs/helpers:108:18)
    at Object.<anonymous> (/PrairieLearn/node_modules/aws-sdk/lib/core.js:88:1)
    at Module._compile (node:internal/modules/cjs/loader:1191:14)
======================================
false
Trace
    at /PrairieLearn/node_modules/aws-sdk/lib/maintenance_mode_message.js:29:11
    at Object.<anonymous> (/PrairieLearn/node_modules/aws-sdk/lib/maintenance_mode_message.js:34:3)
    at Module._compile (node:internal/modules/cjs/loader:1191:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1245:10)
    at Module.load (node:internal/modules/cjs/loader:1069:32)
    at Function.Module._load (node:internal/modules/cjs/loader:904:12)
    at Module.require (node:internal/modules/cjs/loader:1093:19)
    at Module.Hook._require.Module.require (/PrairieLearn/node_modules/require-in-the-middle/index.js:101:39)
    at require (node:internal/modules/cjs/helpers:108:18)
    at Object.<anonymous> (/PrairieLearn/node_modules/aws-sdk/lib/core.js:88:1)
======================================
(node:138) NOTE: We are formalizing our plans to enter AWS SDK for JavaScript (v2) into maintenance mode in 2023.

Please migrate your code to use AWS SDK for JavaScript (v3).
For more information, check the migration guide at https://a.co/7PzMCcy
(Use `node --trace-warnings ...` to show where the warning was created)
info: Workspace server listening on port 8081
info: Workspace host ready
(node:132) NOTE: We are formalizing our plans to enter AWS SDK for JavaScript (v2) into maintenance mode in 2023.

Please migrate your code to use AWS SDK for JavaScript (v3).
For more information, check the migration guide at https://a.co/7PzMCcy
(Use `node --trace-warnings ...` to show where the warning was created)
NOTE: We are formalizing our plans to enter AWS SDK for JavaScript (v2) into maintenance mode in 2023.

Please migrate your code to use AWS SDK for JavaScript (v3).
For more information, check the migration guide at https://a.co/7PzMCcy
    at emitWarning (/PrairieLearn/node_modules/aws-sdk/lib/maintenance_mode_message.js:21:13)
    at /PrairieLearn/node_modules/aws-sdk/lib/maintenance_mode_message.js:32:5
    at Object.<anonymous> (/PrairieLearn/node_modules/aws-sdk/lib/maintenance_mode_message.js:34:3)
    at Module._compile (node:internal/modules/cjs/loader:1191:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1245:10)
    at Module.load (node:internal/modules/cjs/loader:1069:32)
    at Function.Module._load (node:internal/modules/cjs/loader:904:12)
    at Module.require (node:internal/modules/cjs/loader:1093:19)
    at Module.Hook._require.Module.require (/PrairieLearn/node_modules/require-in-the-middle/index.js:101:39)
    at require (node:internal/modules/cjs/helpers:108:18)
info: PrairieLearn server ready, press Control-C to quit
info: Go to http://localhost:3000

```

**Comment by SethPoulsen** [comment]
Wait a minute, I just realized something about my set up was off. Let me try again.

**Comment by jonatanschroeder** [comment]
> Wait a minute, I just realized something about my set up was off. Let me try again.

You may need to run `yarn` again.

**Comment by SethPoulsen** [comment]
Got it figured out. Looks good!

**Review by SethPoulsen — APPROVED** [review]

**Comment by trombonekenny** [comment]
Seems reasonable to me as long as we're tracking the v3 library conversion somewhere else (issue?). It's not the thing we need to show each local docker content developer, though.

**Comment by jonatanschroeder** [comment]
> Seems reasonable to me as long as we're tracking the v3 library conversion somewhere else (issue?). It's not the thing we need to show each local docker content developer, though.

My understanding is that @nwalters512 is tracking this somewhere (note that the comment linked above is his).
